services/ui-src/src/routes/Home/Home.tsx

```diff
@@ -26,6 +26,9 @@ export const Home = () => {
       </Collapse>
       <BasicPage sx={sx.layout} data-testid="home-view">
         <Box sx={sx.introTextBox}>
+          <Heading as="h2" sx={sx.subHeaderText}>
+            {intro.subheader}
+          </Heading>
```

Comment:
This eyebrow header is actually going away per a conversation with design last week. The ticket should have been updated but wasn't. 
```suggestion
```